pkg/controller/replication/replication_controller.go

```diff
@@ -311,6 +256,7 @@ func (rm *ReplicationManager) updatePod(old, cur interface{}) {
 		return
 	}
 	glog.V(4).Infof("Pod %s updated, objectMeta %+v -> %+v.", curPod.Name, oldPod.ObjectMeta, curPod.ObjectMeta)
```

Comment:
Basically we shouldn't log these if the pod does not belong to a RC at all. I have been debugging a statefulset failure and it's pretty annoying that this comes up frequently in logs.